include/sdf/Error.hh

```diff
@@ -179,6 +179,10 @@ namespace sdf
 
     /// \brief Generic error during parsing.
     PARSING_ERROR,
+
+    /// \brief The joint axis mimic does not refer to a valid joint in the
+    /// current scope.
+    JOINT_AXIS_MIMIC_INVALID,
```

Comment:
add this error to `pyError.py`

Reply:
https://github.com/gazebosim/sdformat/pull/1166/commits/93122a3d39f0288235788c70a32a2061e8aac64b